Can't link the parallel runner with multiple translation units

## Expected Behavior

I'd like to configure a parallel runner for a multi-translation unit program. Just took the example `parallel_runner.cpp`, fit into my `main.cpp` and expected the project to just build.

## Actual Behavior

Tons of linker errors

## Steps to Reproduce the Problem

  1. Take `parallel_runner.cpp`
  2. Create another `test.cpp` with tests
  3. Compile and try to link them together
  4. Observe the messages like the following:
```
[build] : && /usr/bin/c++ -g  CMakeFiles/aoc.dir/main.cpp.o CMakeFiles/aoc.dir/test2.cpp.o -o aoc  -lpthread  -lssl  -lcrypto  -lgmp  -ltbb && :
[build] /usr/bin/ld: /usr/bin/ld: DWARF error: could not find variable specification at offset befe
[build] CMakeFiles/aoc.dir/test2.cpp.o:(.bss._ZN5boost3ext2ut6v1_1_83cfgINS2_8overrideEJEEE[_ZN5boost3ext2ut6v1_1_83cfgINS2_8overrideEJEEE]+0x0): multiple definition of `boost::ext::ut::v1_1_8::cfg<boost::ext::ut::v1_1_8::override>'; /usr/bin/ld: DWARF error: could not find variable specification at offset c578
[build] CMakeFiles/aoc.dir/main.cpp.o:(.bss+0x0): first defined here
[build] collect2: error: ld returned 1 exit status
```

## Specifications

  - Version: v1.1.8
  - Platform: Arch Linux x64
  - Subsystem:

--- a/example/cfg/parallel_runner.cpp
+++ b/example/cfg/parallel_runner.cpp
@@ -41,8 +41,8 @@ class parallel_runner : public ut::runner<> {
 };
 }  // namespace cfg
 
-template <>
-auto ut::cfg<ut::override> = cfg::parallel_runner{};
+template <class... Ts>
+inline auto ut::cfg<ut::override, Ts...> = cfg::parallel_runner{};
 
 ut::suite parallel_1 = [] {
   using namespace ut;

